pdc_client/__init__.py

```diff
@@ -185,12 +186,45 @@ def obtain_token(self):
                     raise
         raise Exception('Could not obtain token from any known URL.')
 
+    def __call__(self, *args, **kwargs):
+        return self.client(*args, **kwargs)
+
+    def __getattr__(self, name):
+        return self.client.__getattr__(name)
+
+    def __getitem__(self, *args, **kwargs):
+        return self.client.__getitem__(*args, **kwargs)
+
+    def set_comment(self, comment):
+        """Set PDC Change comment to be stored on the server.
+
+        Once you set the comment, it will be sent in all subsequent requests.
+
+        :param comment:     what comment to send to the server
+        :paramtype comment: string
+        """
+        self.session.headers["PDC-Change-Comment"] = comment
+
+
+class PDCClientWithPage(PDCClient):
+    """
+    PDCClient wrapper specialized for setting page in get_paged function.
+    """
+    def __init__(self, server, token=None, develop=None, ssl_verify=None, page_size=None, page=None):
+        """
+        Create new client instance with page prarameter.
+        Other params are all used for base class.
+        :param page:  the page number of the data.
+        """
+        super(PDCClientWithPage, self).__init__(server, token, develop, ssl_verify, page_size)
+        self.page = page
+
     def get_paged(self, res, **kwargs):
```

Comment:
Removing a public method from `PDCClient` means that next API version should be at least 1.8.x.

But I think it's better to keep it there and just mark it as deprecated for now (and point to a new interface).

Reply:
Do you mean like these codes in class PDCClient?
```
def get_paged(self, res, **kwargs):
    # obsolete
    pass
```

Reply:
Rollback the get_paged codes in PDCClient, and add new codes with self.page in new PDCClientWithPage. Please review